### DISCUSSION TITLE
Any plans to support mobile clients?

### DISCUSSION THREAD

**Discussion by kawpii** [plausible/analytics#1665] [Anything Else]
Since currently Plausible only support web, any plans to support iOS, Android so all analytics data can be seen in one place?

**Comment by metmarkosaric** [comment]
hi! we have [the events API](https://plausible.io/docs/events-api) that allows you to track Android and iOS apps. there's also the Flutter plugin for Plausible: https://pub.dev/packages/plausible_analytics

**Reply by kawpii** [reply]
Thanks, Why there is no native client libraries for mobile? Any plans to support them?

**Reply by metmarkosaric** [reply]
no short term plans unfortunately. we're a small team with limited development resources and we're still working on improving our web analytics product

**Reply by kawpii** [reply]
@metmarkosaric understandable! Any plans to add features like segmentation to the web product?

**Reply by metmarkosaric** [reply]
what do you mean by segmentation? you can apply filters to segment your audience in whatever way you wish already

**Reply by kawpii** [reply]
Yes like applying filters which Plausible seems to support, but there seem to be no way visualizing multiple groups at once ex: two sets of custom filters.

**Reply by metmarkosaric** [reply]
I see. So you'd like the ability to filter say by a custom made group of referral sources rather than by one source at the time or?

**Reply by kawpii** [reply]
What I meant is something like [this](https://lh3.googleusercontent.com/jQiIUJ7nB2HUr45TfwO1RmEYmd7RvX1teSqxeWJ_omDJQypGni5j52CXRyQUN_0_XA=w1400)

**Reply by metmarkosaric** [reply]
cool, thanks for sharing! 

at the moment, you can segment by those who have converted on a specific custom event but would be nice to also be able only show those that haven't converted. there's a thread for that here: https://github.com/plausible/analytics/discussions/1451

any being able to compare different segments in one view would be great to have too. there's a thread on cohorts here: https://github.com/plausible/analytics/discussions/364

we'd like to build these but no promises on either of theme. we are focused on more frequently requested features at the moment such as the GA import. thanks!
